.goreleaser.yml

```diff
@@ -14,9 +14,9 @@ builds:
     goos:
       - darwin
       - linux
+      - windows
     goarch:
       - amd64
-      - arm
```

Comment:
why remove?

Reply:
https://github.com/tendermint/tendermint/blob/909da42789f65018f4024967eee432cb146a97de/libs/math/fraction.go#L49-L49

This line overflows at compile time. It needs to be a string for 32bit. We decided to not touch the code and removed arm from the release.

Reply:
we should change the code if it's easy enough.